lib/src/screen/dice/dice_screen.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+
+import '../../utils/constants.dart';
+
+class DiceScreen extends StatefulWidget {
+  DiceScreen({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _DiceScreenState createState() => _DiceScreenState();
+}
+
+class _DiceScreenState extends State<DiceScreen> {
+
+  @override
+  Widget build(
+    BuildContext context,
+  ) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          widget.title,
+        ),
+      ),
+      body: ListView.builder(
+        itemCount: Constants.diceAmount ~/ 2,
+        itemBuilder: (
+          BuildContext context,
+          int index,
+        ) {
+          Widget bottomMessage = Text('');
+          if (_ifIsLastRow(index)) {
+            bottomMessage = Column(
+              children: <Widget>[
+                Text(
+                  'Tap on any dice to make it roll!',
+                  style: TextStyle(
+                    fontSize: 16.0,
+                    fontWeight: FontWeight.w700,
+                    height: Constants.textHeight1_4,
+                  ),
+                ),
+                Text('Your score is',
+                    style: TextStyle(
+                      fontSize: 26.0,
+                      fontWeight: Constants.fontWeight400,
+                      height: Constants.textHeight1_4,
+                    )),
+                Text('21',
+                    style: TextStyle(
+                      fontSize: 35.0,
+                      fontWeight: Constants.fontWeight400,
+                      height: Constants.textHeight1_4,
+                    )),
+              ],
+            );
```

Comment:
Try to move this outside of the ListView

Reply:
Texts moved to bottomNavigationBar